turbopack/crates/turbopack-dev-server/src/source/headers.rs

```diff
@@ -10,9 +10,17 @@ use turbo_tasks::{NonLocalValue, trace::TraceRawVcs};
 #[serde(transparent)]
 pub struct Headers(BTreeMap<String, HeaderValue>);
 
+impl TaskInput for Headers {
```

Comment:
Why can't this be derived? Because of the BTreeMap?

Reply:
yep.   we could create a generic implementation for BTreeMap, but in this case it seemed obviously safe.   Do you think that is what we should do?